web/src/api/session.ts

```diff
@@ -0,0 +1,55 @@
+import openapiCreateClient from 'openapi-fetch';
+import { paths } from './schema';
+import { createLocalStorageSignal } from '../utils/localstorage';
+import { None, someIfDefined } from '../types';
+import { createMemo } from 'solid-js';
+
+const createClient = (options: RequestInit) =>
```

Comment:
I would make the `option` parameter optional:

```ts
const createClient = (options?: RequestInit = {}) => {}; // or
const createClient = (options: Option<RequestInit>) => {};
```

In the following code we then can use:

```ts
const client = createClient(); // or
const client = createClient(None);
```

Reply:
b219db2896ebce6806b93a06b58b2d14630dfe5a